synths/core-synths.scd

```diff
@@ -329,6 +329,24 @@ live coding them requires that you have your SuperDirt instance in an environmen
 		ReplaceOut.ar(out, signal);
 	}, [\ir, \ir, \ir]).add;
 
+	// Phaser audio DSP effect definition. Params: phaserrate (Hz); phaserdepth (amplitude)
+	SynthDef("dirt_phaser" ++ numChannels, { |out, phaserrate = 1.0, phaserdepth = 0.5 |
+		var signal, phaserwave;
+		var maxDelayTime = 4;
```

Comment:
you can do one thing: add `sustain` as an argument to the synth, and use that value as a `maxDelayTime`. Saves *a lot* of memory!